package.json

```diff
@@ -65,7 +65,7 @@
     "webmaker-translation-stats": "0.0.1"
   },
   "scripts": {
-    "postinstall": "node_modules/.bin/bower install && node_modules/.bin/grunt requirejs:dist",
+    "postinstall": "node_modules/.bin/bower install && grunt requirejs:dist",
```

Comment:
@humphd then it would run the globally installed `bower` and `grunt`. Like I mentioned on the issue, they are ran on the same scope.

maybe `cd node_modules/.bin/ && bower install && grunt requirejs` would be clearer?